blockbase/package.json

```diff
@@ -3,37 +3,26 @@
   "version": "2.0.9",
   "description": "Blank Parent Theme",
   "bugs": {
-    "url": "https://github.com/Automattic/themes/issues"
+    "url": "https://github.com/Automattic/themes/issues/new?label=[Theme]%20Blockbase"
   },
-  "homepage": "https://github.com/Automattic/themes#readme",
+  "homepage": "https://github.com/Automattic/themes/blob/trunk/blockbase/README.md",
   "devDependencies": {
     "@wordpress/base-styles": "^3.3.0",
     "@wordpress/browserslist-config": "^2.2.2",
     "chokidar-cli": "^2.1.0",
-    "deepmerge": "^4.2.2",
     "node-sass": "^4.13.1",
     "node-sass-package-importer": "^5.3.2",
-    "npm-run-all": "^4.1.5",
-    "rtlcss": "^2.4.0"
-  },
-  "rtlcssConfig": {
-    "options": {
-      "autoRename": false,
-      "autoRenameStrict": false,
-      "blacklist": {},
-      "clean": true,
-      "greedy": false,
-      "processUrls": false,
-      "stringMap": []
-    },
-    "plugins": [],
-    "map": false
+    "postcss": "^8.2.12",
+    "postcss-cli": "^8.3.1"
+
   },
   "browserslist": [
     "extends @wordpress/browserslist-config"
   ],
   "scripts": {
     "start": "chokidar \"sass/**/*.scss\" -c \"npm run build\" --initial",
-    "build": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js sass/ponyfill.scss assets/ponyfill.css --output-style expanded --indent-type tab --indent-width 1 --source-map true"
-  }
+    "build": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js sass/ponyfill.scss assets/ponyfill.css --output-style expanded --indent-type tab --indent-width 1 --source-map true | postcss --use autoprefixer --output assets/theme.css --map"
```

Comment:
when I build I get an error now:
```

CssSyntaxError: /Users/maggie/A8c/repos/a8c-projects/themes/blockbase/assets/theme.css:1:1: Unknown word

> 1 | Rendering Complete, saving .css file...
    | ^
  2 | Wrote Source Map to /Users/maggie/A8c/repos/a8c-projects/themes/blockbase/assets/ponyfill.css.map
  3 | Wrote CSS to /Users/maggie/A8c/repos/a8c-projects/themes/blockbase/assets/ponyfill.css
```

Reply:
When that's fixed, building the css will probably mean a few changes because of the autoprefixer, so we should probably include those changes here too

Reply:
Try now!